api/queries_repo.go

```diff
@@ -235,6 +238,13 @@ func (r Repository) ViewerCanTriage() bool {
 	}
 }
 
+// MergeQueue is required for this repository and branch
+func (r Repository) MergeQueueRequired() bool {
```

Comment:
I wasn't sure if this is the right place for this method. A merge queue is a branch protection feature that is required for a given repository and a target branch. Having this on Repository isn't exactly correct, but it seemed to be a good way to avoid exposing too much of the internals of the api response to the caller